src/poprox_storage/repositories/demographics.py

```diff
@@ -30,9 +30,10 @@ def store_demographics(self, demographic: Demographics) -> None:
                 account_id=demographic.account_id,
                 gender=demographic.gender,
                 birth_year=demographic.birth_year,
-                zip5=demographic.zip5,
+                zip3=demographic.zip5,
```

Comment:
I think this will work but we should also change the name of the field to `zip3` here

Reply:
Do we need an explicit truncation here?

Reply:
It's confusing because of the attribute name (which needs to change) but `Demographics` only has `zip3` so both sides of the `=` are already `zip3` here